Images::getInfoFromURL[Cached]() will both return empty arrays ...

Quix0r opened the pull request:
... on error and that needs to be reflected here, too.

Follow-up PR to https://github.com/friendica/friendica/pull/11673

Quix0r committed:
Let's avoid local variables

Quix0r committed:
Images::getInfoFromURL[Cached]() will both return empty arrays on error and that needs to be reflected here, too.

MrPetovan commented on statusnet/statusnet.php:
```diff
@@ -1050,7 +1050,7 @@ function statusnet_createpost(App $a, $uid, $post, $self, $create_user, $only_ex
 
 		if ($content->user->id == $own_url) {
-			$self = DBA::selectFirst([], ['self' => true, 'uid' => $uid]);
+			$self = DBA::selectFirst('', ['self' => true, 'uid' => $uid]);
```
This call is missing a table name. The empty array means `*` for the field selection parameter.

Quix0r replied:
I added the `*` there.

MrPetovan commented on twitter/twitter.php:
```diff
@@ -1279,7 +1279,7 @@ function twitter_fix_avatar($avatar)
 
 	$info = Images::getInfoFromURLCached($new_avatar);
-	if (!$info) {
+	if (empty($info)) {
```
`empty()` is superfluous as empty array is natively cast as false in a condition.

Quix0r replied:
Once we have decided that in main repository's PR, I will change it here as well. :-)

Quix0r committed:
Converted more double-quotes to single

Quix0r committed:
Converted more double-quotes to single

MrPetovan commented on statusnet/statusnet.php:
```diff
@@ -1050,7 +1050,7 @@ function statusnet_createpost(App $a, $uid, $post, $self, $create_user, $only_ex
 
 		if ($content->user->id == $own_url) {
-			$self = DBA::selectFirst('', ['self' => true, 'uid' => $uid]);
+			$self = DBA::selectFirst('*', ['self' => true, 'uid' => $uid]);
```
No, I mean, it needs a table name as first argument, and then an array of field names which can be empty, meaning `*`.
```suggestion
			$self = DBA::selectFirst('table', [], ['self' => true, 'uid' => $uid]);
```
Not sure which table it is though but it looks like `contact`.

Quix0r committed:
Changed more double-quotes to single

Quix0r committed:
Changes:
- ops, my bad that I misunderstood @MrPetovan here and maybe me having a cold :-/
- also made $contactId camel-case
